src/solve.h

```diff
@@ -13,6 +13,7 @@
 #define SOLVE_H
 
 #include "sat_types.h"
+#include <malloc.h>
```

Comment:
I don't understand the question.
Do you mean what benefits do we get from explicitly including malloc.h?
It just kills a warning I was getting. The compiler would've done it automatically, but it technically IS in another header.